src/js/index.js

```diff
@@ -136,6 +153,12 @@ export const clearScreen = function(){
 
 
 export const setUI = async function(state, user){
+    state.cleanseEvent = function(className){
+        const oldNavigator = document.querySelector(className);
+        const newNavigator = oldNavigator.cloneNode(true);
+        oldNavigator.parentNode.replaceChild(newNavigator, oldNavigator);
+    };
```

Comment:
I don't think this should be associated with state, a global function would probably make more sense

Reply:
Done